### ISSUE TITLE
Patterns into prototype: Request alternative formats plugin

### ISSUE THREAD

**Issue by martinwake** [dwp/design-system#656]
## What
Review and update the documentation for the Alternative formats plugin, then announce it in the same way as the Find an address plugin.

## Why
The Find an address plugin has given us a model for how to do an "official" release of a plugin and we should bring the alt formats plugin up to the same standard. This will raise awareness of the plugin and the pattern and drive adoption.

## Done when

- [x] Documentation tidied up to match example of address plugin
- [x] Technical checks and any necessary changes (eg renaming repo)
- [x] Announced on Slack channels
- [ ] Latest updates page on website

## Outcomes
The plugin is available and publicised, and probably included in the standard repo for interaction designers.

## Who needs to know about this
- Design System team
- IxD practice
- Content design practice
- Front end devs
- SRA / Customer preference service

## Related stories
#635
